content/guides/references/configuration.md

```diff
@@ -207,7 +207,7 @@ object:
 | Option            | Default                               | Description                                                                                                                                                                                         |
 | ----------------- | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `setupNodeEvents` | `null`                                | Function in which node events can be registered and config can be modified. Takes the place of the (deprecated) plugins file. [Please read the notes for examples on using this.](#setupNodeEvents) |
-| `supportFile`     | `cypress/support/index.js`            | Path to file to load before test files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                           |
+| `supportFile`     | `cypress/support/e2e.js`              | Path to file to load before test files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                           |
```

Comment:
The default files have been updated to include more extensions, see https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#8399c9d90abd416eace7c3f4b2aeb84a.

Also, we need to be very clear that for a given testing type, the user can only have one matching support file, multiple matching support files will cause an error. https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#88cb8241a661447da93a0bf47f86d9f7